src/cursor/cur_index.c

```diff
@@ -349,13 +349,15 @@ static int
 __curindex_open_colgroups(
     WT_SESSION_IMPL *session, WT_CURSOR_INDEX *cindex, const char *cfg_arg[])
 {
+	WT_CONFIG_ITEM cval;
 	WT_TABLE *table;
 	WT_CURSOR **cp;
-	u_long arg;
+	char cfg_bulk[30], *proj;
 	/* Child cursors are opened with dump disabled. */
-	const char *cfg[] = { cfg_arg[0], cfg_arg[1], "dump=\"\"", NULL };
```

Comment:
@ddanderson Is this change necessary? I'm surprised you need to open a column group with ```bulk=unordered```. I ran the set of schema tests with this part of the change backed out and everything passed.